Gemfile

```diff
@@ -9,6 +9,10 @@ gem 'jquery-rails'
 gem 'jbuilder', '~> 2.0'
 gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'feedjira'
+gem 'responders'
+gem 'active_model_serializers'
+gem 'will_paginate'
```

Comment:
I think we already have this gem in master now that @Kealii's code has been merged